docs/scos/user/features/202204.0/configurable-product-feature-overview.md

```diff
@@ -131,7 +131,7 @@ If a developer defines that the configuration of such a product needs to be comp
 
 Customers configure a product on the Storefront as follows:
 
-![configurable-product-on-the-storefront](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Configurable+Product/Configurable+Product+feature+overview/configurable-product-on-the-storefront.gif)
+<iframe width="960" height="720" src="https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/features/configurable-product-feature-overview/configurable-product-on-the-storefront.mp4" frameborder="0" allowfullscreen></iframe>
```

Comment:
Please cut out top and bottom parts of the video. Only the page should be visible on the video

Reply:
@andriitserkovnyi done